lib/src/main/java/com/futurewei/alcor/common/utils/Ipv4AddrUtil.java

```diff
@@ -63,4 +68,13 @@ public static String longToIpv4(long longIpv4) {
 
         return result;
     }
+
+    public static boolean ipv4PrefixCheck(String strIpv4) {
```

Comment:
Like it!

Reply:
Let us get some UTs to cover this basic util method.

Reply:
no problem